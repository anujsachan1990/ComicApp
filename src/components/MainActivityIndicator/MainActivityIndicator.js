import React, { memo } from 'react'

import { ActivityIndicator, StyleSheet, View } from 'react-native'

const styles = StyleSheet.create({
  activityIndicatorContainer: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'center',
  },
})

const mainActivityIndicator = () => (
  <View style={styles.activityIndicatorContainer}>
    <ActivityIndicator />
  </View>
)

export default memo(mainActivityIndicator)
