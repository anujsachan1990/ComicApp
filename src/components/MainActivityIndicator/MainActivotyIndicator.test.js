import React from 'react'
import renderer from 'react-test-renderer'

import MainActivityIndicator from './MainActivityIndicator'

describe('<MainActivityIndicator />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<MainActivityIndicator />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
