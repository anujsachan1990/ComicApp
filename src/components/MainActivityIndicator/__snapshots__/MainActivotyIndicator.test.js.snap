// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainActivityIndicator /> renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
  />
</View>
`;
